### PR TITLE
DeviceDateTimeManagement.set - optional ntpServer field added

### PR DESCRIPTION
**Pull request by michal-signagelab-com**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [1.7.0] - 2021-05-19
```
Comment (misak113) [inline CHANGELOG.md:8]: Just please change it to following (the  specific version and date will be added automatically during release process)
```md
## [Unreleased]
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -5,6 +5,10 @@ All notable changes to this project will be documented in this file.
...
+### Added
```
Comment (misak113) [inline CHANGELOG.md:9]: I'd add it to Fixed section. It had to be there already.

File: CHANGELOG.md (hunk 1 above)
Comment (michal-signagelab-com) [reply CHANGELOG.md]: Fixed

File: CHANGELOG.md (hunk 2 above)
Comment (michal-signagelab-com) [reply CHANGELOG.md]: Fixed

File: tests/Unit/RestApi/Device/DateTime/DeviceDateTimeManagement.spec.ts [hunk 3]
```diff
@@ -18,6 +18,7 @@ describe('DeviceDateTimeManagement', () => {
 	const validSetReq: IDeviceDateTimeUpdatable = {
 		time: '2018-09-09T09:09:34.001',
 		timezone: 'Europe/Prague',
+		ntpServer: 'pool.ntp.org'
```
Comment (misak113) [inline tests/Unit/RestApi/Device/DateTime/DeviceDateTimeManagement.spec.ts:21]: tak jeste posledni.. tady chybi trailing comma. muzes pustit `npm run lint`

Comment (michal-signagelab-com) [reply tests/Unit/RestApi/Device/DateTime/DeviceDateTimeManagement.spec.ts]: Fixed

**Pull request merged**